src/main/java/cultfit/FitnessCentre.java

```diff
@@ -0,0 +1,86 @@
+package cultfit;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FitnessCentre {
+    private List<User> users;
+    private List<FitnessProgram> programs;
```

Comment:
users cannot be in fitness center. what happns in case of multiple branches.

Reply:
A fitness centre is a branch itself which will have access to a list of users.

Reply:
access users from a different class like UserService